Apps/Src/Task_PetWDog.c

```diff
@@ -14,7 +14,10 @@ void Task_PetWDog(void *p_arg) {
     // If WDog_BitMap is all set:
     //      Reset watchdog timer
 
-
+     if (WDog_BitMap == 7) {     
```

Comment:
Not sure how this change got in here, since I thought we already merged this into the rtos branch